test/hummingbot/connector/gateway/amm/debug_gateway_evm_amm.py

```diff
@@ -173,7 +173,7 @@ def create_order_record(
                 gas_price=gas_price,
                 creation_timestamp=self._connector.current_timestamp
             )
-        print("Getting uniswap order status...\t\t", end="", flush=True)
+        print("Getting amm order status...\t\t", end="", flush=True)
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed